bin/fuel-core/chainspec/local-testnet/state_config.json

```diff
@@ -119,6 +119,7 @@
     }
   ],
   "messages": [],
+  "blobs": [],
```

Comment:
We shouldn't break state config. The `blobs` should be `Option`

Reply:
`blobs` is `#[serde(default)]`. Isn't this enough?

Reply:
tested with a local node connected to testnet that serde(default) works 👍🏻